grammar.js

```diff
@@ -421,13 +431,19 @@ module.exports = grammar({
       ))),
       seq(":'", $._single_quoted_continuation),
       seq(':"', $._double_quoted_continuation),
+      ':"#"', // TODO: Remove this hack
```

Comment:
Sorry this isn't clear and was still editing the PR body. This and the other places where we have `#` and the `TODO: Remove this hack` is for parsing a single `#` in rules that allow interpolation. The underlying tree-sitter issue is that we can't do something like `/#[^{]/|#$/` and match either `#` not followed by `{` or `#` if it's at the end of the line.